### ISSUE TITLE
Extensions in Generated StructureDefinitions are wrong 

### ISSUE THREAD

**Issue by markkramerus** [FHIR/sushi#116]
There is a section of extension within each SD generated by SUSHI that includes the standards status, fhir maturity (fmm), and work group. The values given are wrong, and should be corrected or the extension removed:

* The Standards Status is "normative" -- this is false. It should be derived from the IG json or package.json file, in mCODE's case, "trial-use" would be appropriate (note: the value must come from this value set: http://hl7.org/fhir/valueset-standards-status.html).

* The maturity (fmm) is set at level 5.  This level is the highest maturity level, and should be assumed. A user can set it in FSH using the escape (^) syntax, but otherwise, it probably should be omitted entirely.

* If the maturity is specified, the "Text Summary" of each Profile page includes the text "Maturity: 5".  with a link to a non-existent URL, `versions.html#maturity`. I am not sure what assumption the IG Publisher is making here, but we can't have a 404 happening here. 

* The Work group (wg) extension is specified, and it seems to be the work group responsible for the resource, instead of the work group responsible for the profile. It should be the latter. The work group responsible for the profile is in the IG json or maybe package.json.

**Comment by cmoesel** [comment]
These are all copied from the standard resource before we begin profiling it.  You are correct, these extensions should be removed or updated.  I would suggest we _remove_ them entirely.  Authors can add them back in with appropriate values using the `^` syntax and/or we can discuss and design mechanisms for automatically setting them in future versions of SUSHI.

**Comment by markkramerus** [comment]
Another key piece of information:  The QA report is saying these extensions are actually illegal in the place they are used:
```
StructureDefinition/ComorbidCondition: StructureDefinition.snapshot.element[1].type[0] | error | The extension http://hl7.org/fhir/StructureDefinition/structuredefinition-fhir-type is not allowed to be used at this point (allowed = e:ElementDefinition.type.code)
-- | -- | --
StructureDefinition/ComorbidCondition: StructureDefinition.snapshot.element[16].type[0] | error | The extension http://hl7.org/fhir/StructureDefinition/structuredefinition-fhir-type is not allowed to be used at this point (allowed = e:ElementDefinition.type.code)
StructureDefinition/ComorbidCondition: StructureDefinition.snapshot.element[31].type[0] | error | The extension http://hl7.org/fhir/StructureDefinition/structuredefinition-fhir-type is not allowed to be used at this point (allowed = e:ElementDefinition.type.code)
StructureDefinition/ComorbidCondition: StructureDefinition.snapshot.element[38].type[0] | error | The extension http://hl7.org/fhir/StructureDefinition/structuredefinition-fhir-type is not allowed to be used at this point (allowed = e:ElementDefinition.type.code)
```

**Comment by cmoesel** [comment]
Fixed in 0.6.1

**Comment by cmoesel** [comment]
This approach is superceded by https://jira.hl7.org/browse/FHIR-27535 and will be changed in an upcoming version of SUSHI.
